webapp/src/components/Validation/ValidateInput/ValidateInput.js

```diff
@@ -13,6 +13,9 @@ const validateInput = (value, inputValidations = {}, type = "text") => {
   const emailRegex = new RegExp(
     /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
   );
+
```

Comment:
@vishaltambe70 these changes are not required since we are not making any chnages in this branch for validations.please revert it.